src/styles/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  box-sizing: border-box;
+  margin: auto;
+  gap: $dayGap;
+  width: getCalendarSize($daySize, $dayGap, $padding);
+  padding: 20px $padding 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
```

Comment:
Maybe here its no need to repeat if you put "box-sizing: border-box" on tag calendar?

Reply:
I've checked, border-box don't inherit